doc/stages/filters.icp.rst

```diff
@@ -128,3 +128,6 @@ rt
 
 tt
   Translation threshold. [Default: **9e-8**]
+
```

Comment:
The file's encoding or something seems wonky here.

Reply:
I see that pretty regularly. No clue.